specification/signalr/resource-manager/Microsoft.SignalR/preview/2018-03-01-preview/signalr.json

```diff
@@ -0,0 +1,1034 @@
+                        }
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "description": "Request accepted. The response contains a Location header to query the operation result."
+                    },
+                    "204": {
+                        "description": "No content. The response indicates no response is eligible to move."
+                    }
+                },
+                "x-ms-long-running-operation": true,
+                "x-ms-examples": {
+                    "MoveResources": {
+                        "$ref": "./examples/MoveResources.json"
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/validateMoveResources": {
+            "post": {
+                "tags": [
+                    "SignalR"
+                ],
+                "description": "Validate the resource can be moved or not",
+                "operationId": "SignalR_ValidateMoveResources",
+                "parameters": [
+                    {
+                        "name": "moveResourcesRequest",
+                        "in": "body",
+                        "description": "parameters describes the move operation.",
+                        "required": false,
+                        "schema": {
+                            "$ref": "#/definitions/MoveResourcesRequest"
+                        }
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success. The resources can be moved."
+                    },
+                    "400": {
+                        "description": "Bad request. The resources cannot be moved."
+                    }
+                },
+                "x-ms-examples": {
+                    "ValidateMoveResources": {
+                        "$ref": "./examples/ValidateMoveResources.json"
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalR/SignalR/{resourceName}/keys": {
+            "get": {
```

Comment:
I don't think you can return keys using `GET` in ARM due to RBAC. Please change this to a `POST`.